stage1/modules/media-queries/README.md

```diff
@@ -1,15 +1,17 @@
-# [Stage#1](../../) Медиазапросы
-## В данном модуле студентам необходимо познакомиться:
-- изучить основы медиазапросов
-- знать основные операторы
-- знать назначение метатэга viewport
+# [Stage#1](../../) Media Queries
+## In this module students should:
+- learn the basics of media queries
+- understand key operators
+- understand the purpose of the viewport meta tag.
 
-## Приблизительное время прохождения модуля
-2 часа
+## Estimated Module Duration
+2,5 hours
 
-## Теория 
-1. [CSS3-медиазапросы](https://html5book.ru/css3-mediazaprosy/) - 30 мин
-2. [Уроки CSS - Медиа запросы основы](https://www.youtube.com/watch?v=M-xc1EOMOIE) - 30 мин"
+## Theory 
+1. [A Complete Guide to CSS Media Queries](https://css-tricks.com/a-complete-guide-to-css-media-queries/) - 30 minutes
+2. [Viewport meta tag](https://developer.mozilla.org/en-US/docs/Web/HTML/Viewport_meta_tag) - 15 minutes
+3. [Learn CSS Media Query in 7 minutes](https://www.youtube.com/watch?v=yU7jJ3NbPdA) - 7 minutes
+4. [Make a website responsive for all devices](https://www.youtube.com/watch?v=UUjNEMXZA-k) - 30 minutes
 
-## Практика   
-1. Вам необходимо пройти тест "Media Queries & Responsive" в RS APP > Auto Test
+## Practice   
+1. You need to complete the "Media Queries & Responsive" test in RS APP > Auto Test.
```

Comment:
`You are required to complete the "Media Queries & Responsive" test in RS APP > Auto Test.`